python/src/sdf/pySurface.cc

```diff
@@ -55,6 +106,11 @@ void defineSurface(pybind11::object module)
          "Get the associated contact object")
     .def("set_contact", &sdf::Surface::SetContact,
          "Set the associated contact object.")
+    .def("friction", &sdf::Surface::Friction,
+         pybind11::return_value_policy::reference,
```

Comment:
```suggestion
         pybind11::return_value_policy::reference_internal,
```

Reply:
should the `contact` and `ode` APIs in this file also use `reference_internal`?

Reply:
https://github.com/gazebosim/sdformat/pull/1036/commits/5da18f0d034a3bfbcb0e3096ab21f306b5352d96